### ISSUE TITLE
[BUG] “Content settings unavailable”

### ISSUE THREAD

**Issue by henderea** [Paperback-iOS/app#364]
**Describe the bug**
In general settings in 0.6.0, the content filtering section says "content settings unavailable", and when looking in the source repos, I see a message that says X results are filtered out, with no results actually showing up. 

**To Reproduce**
Look in general settings and a source repo

**Expected behaviour**
There should be some sort of content settings and I should see sources in the repos. 

**App Version**
V0.6.0-1.0.2

**Screenshots**
![image](https://user-images.githubusercontent.com/657766/124627212-5d0e5300-de4d-11eb-9cdb-9cd3da116ae1.png)
![image](https://user-images.githubusercontent.com/657766/124627227-60a1da00-de4d-11eb-90a2-aed9acb7e35c.png)

**Comment by TheNetsky** [comment]
Yoinked from our support bot, should solve the issue
![image](https://user-images.githubusercontent.com/56271887/124627391-cc049f80-de7f-11eb-9d4f-47b726a975e6.png)

**Comment by henderea** [comment]
Thanks, that fixes the issue. Might be good to have a button in the app to launch the portal.

**Comment by TheNetsky** [comment]
That's not allowed by Apple guidelines, so we can't do that.

**Comment by henderea** [comment]
Too bad. Oh well. Thanks for the quick answers.

Oh, and I should point out that even the official "paperback" repo was filtered out before I went to the portal, so you might want to fix that. The second screenshot I posted was from the "official repo". I can't imagine the official "paperback" repo is actually one that would need to be filtered out, since the whole point of it is to remove adult content to meet App Store guidelines.
